Documentation/guides/DotNet5.md

```diff
@@ -53,11 +53,12 @@ Default Android related file globbing behavior is defined in `Microsoft.Android.
 
 ## dotnet cli
 
-There are currently two "verbs" we are aiming to get working in
+There are currently a few "verbs" we are aiming to get working in
 Xamarin.Android:
 
     dotnet build
     dotnet publish
+    dotnet run
 
 Currently in .NET Core (aka .NET 5), `dotnet publish` is where all the
```

Comment:
These publish lines may need a bit of rewording as well.

Reply:
This should probably say "in .NET 5 console apps" 👍